curl/curl.yaml

```diff
@@ -4,17 +4,17 @@ metadata:
   name: curl
 spec:
   params:
-   - name: url
-     description: URL to curl'ed
-     type: string
-   - name: options
-     description: options of url
-     type: array
-     default: []
-   - name: curl-image
-     description: option of curl image
-     type: string
-     default: "curlimages/curl"
+    - name: url
```

Comment:
this change isn't strictly required and changes the convention use throughout the catalog. could we please revert this whitespace style change 🙏 ?